docs/migrate-to-express-package.md

```diff
@@ -0,0 +1,145 @@
+# Migrating to `@nestjs-shopify/express`
+
+The `@nestjs-shopify/core` package has been deprecated in favor of `@nestjs-shopify/express` or `@nestjs-shopify/fastify`. The new packages are designed to work with the respective web frameworks and provide a more flexible and powerful API.
+
+To migrate your NestJS app to use the new adaptable version, follow this guide.
+
+## Table of Contents
+
+- [Express](#express-package)
+- [Fastify](#fastify-package)
+- [Changes to CSP](#changes-to-csp)
+- [Changes to HMAC validation](#changes-to-hmac-validation)
+
+## Express package
+
+Add the new `@nestjs-shopify/express` package to your project:
+
+```bash
+npm install @nestjs-shopify/express
+```
+
+or using Yarn:
+
+```bash
+yarn add @nestjs-shopify/express
+```
+
+You will have to replace all occurences to `ShopifyCoreModule` with `ShopifyExpressModule` in your application root module:
+
+```patch
+// app.module.ts
+import { Module } from '@nestjs/common';
+-import { ShopifyCoreModule } from '@nestjs-shopify/core';
++import { ShopifyExpressModule } from '@nestjs-shopify/express';
+
+@Module({
+  imports: [
+-    ShopifyCoreModule.forRoot({
++    ShopifyExpressModule.forRoot({
+      apiKey: 'foo',
+      apiSecret
+```
+
+## Fastify package
+
+Add the new `@nestjs-shopify/fastify` package to your project:
+
+```bash
+npm install @nestjs-shopify/fastify
+```
+
+or using Yarn:
+
+```bash
+yarn add @nestjs-shopify/fastify
+```
+
+You will have to replace all occurences to `ShopifyFastifyModule` with `ShopifyExpressModule` in your application root module:
```

Comment:
If you want to keep this part, either this is wrong:

"You will have to replace all occurences to `ShopifyFastifyModule` with `ShopifyExpressModule` in your application root module:"

or the diff below is wrong, since it replaces "ShopifyCoreModule"